src/pacmod3_node.cpp

```diff
@@ -88,71 +88,71 @@ LNI::CallbackReturn PACMod3Node::on_configure(const lc::State & state)
+    this->create_publisher<pacmod_msgs::msg::BrakeAuxRpt>("brake_aux_rpt", 20);
   can_pubs_[ShiftAuxRptMsg::CAN_ID] =
-    this->create_publisher<pacmod_msgs::msg::ShiftAuxRpt>("parsed_tx/shift_aux_rpt", 20);
+    this->create_publisher<pacmod_msgs::msg::ShiftAuxRpt>("shift_aux_rpt", 20);
   can_pubs_[SteerAuxRptMsg::CAN_ID] =
-    this->create_publisher<pacmod_msgs::msg::SteerAuxRpt>("parsed_tx/steer_aux_rpt", 20);
+    this->create_publisher<pacmod_msgs::msg::SteerAuxRpt>("steering_aux_rpt", 20);
   can_pubs_[TurnAuxRptMsg::CAN_ID] =
-    this->create_publisher<pacmod_msgs::msg::TurnAuxRpt>("parsed_tx/turn_aux_rpt", 20);
+    this->create_publisher<pacmod_msgs::msg::TurnAuxRpt>("turn_aux_rpt", 20);
   can_pubs_[VehicleSpeedRptMsg::CAN_ID] =
-    this->create_publisher<pacmod_msgs::msg::VehicleSpeedRpt>("parsed_tx/vehicle_speed_rpt", 20);
+    this->create_publisher<pacmod_msgs::msg::VehicleSpeedRpt>("vehicle_speed_rpt", 20);
   can_pubs_[VinRptMsg::CAN_ID] =
-    this->create_publisher<pacmod_msgs::msg::VinRpt>("parsed_tx/vin_rpt", 5);
+    this->create_publisher<pacmod_msgs::msg::VinRpt>("vin_rpt", 5);
 
   pub_enabled_ = this->create_publisher<std_msgs::msg::Bool>(
-    "as_tx/enabled", rclcpp::QoS(1).transient_local());
+    "enabled", rclcpp::QoS(1).transient_local());
   pub_vehicle_speed_ms_ = this->create_publisher<std_msgs::msg::Float64>(
-    "as_tx/vehicle_speed", 20);
+    "vehicle_speed", 20);
   pub_all_system_statuses_ = this->create_publisher<pacmod_msgs::msg::AllSystemStatuses>(
-    "as_tx/all_system_statuses", 20);
+    "all_system_statuses", 20);
 
   sub_can_tx_ = this->create_subscription<can_msgs::msg::Frame>(
     "can_tx", 100, std::bind(&PACMod3Node::callback_can_tx, this, std::placeholders::_1));
 
   can_subs_[AccelCmdMsg::CAN_ID] = std::make_pair(
     this->create_subscription<pacmod_msgs::msg::SystemCmdFloat>(
-      "as_rx/accel_cmd", 20,
+      "accel_cmd", 20,
       std::bind(&PACMod3Node::callback_accel_cmd, this, std::placeholders::_1)),
     std::shared_ptr<LockedData>(new LockedData(AccelCmdMsg::DATA_LENGTH)));
 
   can_subs_[BrakeCmdMsg::CAN_ID] = std::make_pair(
     this->create_subscription<pacmod_msgs::msg::SystemCmdFloat>(
-      "as_rx/brake_cmd", 20,
+      "brake_cmd", 20,
       std::bind(&PACMod3Node::callback_brake_cmd, this, std::placeholders::_1)),
     std::shared_ptr<LockedData>(new LockedData(BrakeCmdMsg::DATA_LENGTH)));
 
   can_subs_[ShiftCmdMsg::CAN_ID] = std::make_pair(
     this->create_subscription<pacmod_msgs::msg::SystemCmdInt>(
-      "as_rx/shift_cmd", 20,
+      "shift_cmd", 20,
       std::bind(&PACMod3Node::callback_shift_cmd, this, std::placeholders::_1)),
     std::shared_ptr<LockedData>(new LockedData(ShiftCmdMsg::DATA_LENGTH)));
 
   can_subs_[SteerCmdMsg::CAN_ID] = std::make_pair(
     this->create_subscription<pacmod_msgs::msg::SteerSystemCmd>(
-      "as_rx/steer_cmd", 20,
+      "steer_cmd", 20,
```

Comment:
Since steer_rpt and steer_aux_rpt were changed from steer_ to steering_, should steer_cmd be changed to steering_cmd also for consistency?

Reply:
Absolutely! thanks for catching this, I missed it.